src/components/OfflineWithFeedback.js

```diff
@@ -89,6 +89,8 @@ function applyStrikeThrough(children) {
 
 const OfflineWithFeedback = (props) => {
     const hasErrors = !_.isEmpty(props.errors);
+    const errorMessages = _.omit(props.errors, (e) => e === null);
+    const hasErrorMessages = !_.isEmpty(errorMessages);
```

Comment:
```suggestion
    const hasErrorMessages = hasErrors && _.some(props.errors);
```

Reply:
Why would be pass `null` errors in that case?

Reply:
I wanted to avoid the possibility of blank lines in case at some point in future the object has both valid and `null` errors.

Reply:
Ah right nice catch. Can you just add a comment explaining that we may add errors with null to apply opacity without showing errors

Reply:
Sure, but where should I add it?

Reply:
just above `errorMessages`